### DISCUSSION TITLE
How would I disable land claiming, but keep everything else especially /restorenature?

### DISCUSSION THREAD

**Discussion by CaptainGolem** [GriefPrevention/GriefPrevention#1536] [Q&A] [answered]
How would I disable land claiming, but keep everything else especially /restorenature? I use landlord2 for claims, but the /restore is very nice and I'd like to use it.

**Comment by Jikoo** [comment, marked as answer]
You could try setting claims mode to `Disabled`. If that doesn't work, deny players the permission to use claims (`griefprevention.claims`) in those worlds.

**Comment by CaptainGolem** [comment]
Thanks!
